Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -401,6 +430,31 @@ - (void)scrollToDate:(BPKSimpleDate *)date {
     self.calendarView.pagingEnabled = NO;
 }
 
+- (void)selectWholeMonth:(BPKSimpleDate *)month {
+    self.isSelectingWholeMonth = YES;
+    NSDate *date = [month dateForCalendar:self.gregorian];
+    NSDate *minDate = [self.minDate dateForCalendar:self.gregorian];
+    NSArray<BPKSimpleDate *> *selectedDates = [self.dateProvider dateListForMonth:date fromMinDate:minDate];
+
+    BOOL previousMultiSelectionConfiguration = self.calendarView.allowsMultipleSelection;
+    self.calendarView.allowsMultipleSelection = YES;
```

Comment:
What will allowsMultipleSelection do when we set it to YES in this function? It feels like a bit of a 'hack' considering we restore it at the end of the function.

Reply:
Ok, I've just realised I marked as resolved this question when it wasn't 🤦‍♂️ 
Yes, this is a hack, but it's the only way I could find to mark the whole month as selected when the selection configuration is simple (only one date can be selected), without this hack the only date marked as selected in single selection configuration is the last day of the month.

Reply:
Makes sense to me, should we add a comment indicating that 'selecting whole month' does not take the calendar type into account?

Reply:
Sorry, I just saw this comment right now, after is merged 😵‍💫
But yes, I think it makes sense, too.